content/developer/tutorials/discover_js_framework/02_web_framework.rst

```diff
@@ -94,10 +94,10 @@ services, and components can import a service with the `useService()` hook.
    #. A button `Customers`, which opens a kanban view with all customers (this action already
       exists, so you should use `its xml id
       <https://github.com/odoo/odoo/blob/1f4e583ba20a01f4c44b0a4ada42c4d3bb074273/
-      odoo/addons/base/views/res_partner_views.xml#L525>`_).
+      odoo/addons/base/views/res_partner_views.xml#L510>`_).
    #. A button `New Orders`, which opens a list view with all orders created in the last 7 days. Use
       the `Domain <https://github.com/odoo/odoo/blob/1f4e583ba20a01f4c44b0a4ada42c4d3bb074273/
-      odoo/addons/web/static/src/core/domain.js#L19>`_ helper class to represent the domain.
```

Comment:
This has already been fixed with https://github.com/odoo/documentation/pull/5702